build.gradle

```diff
@@ -15,12 +15,15 @@ plugins {
   id 'com.bmuschko.docker-java-application' version '3.6.2'
   id "nebula.ospackage-application" version "9.1.1"
   id 'org.ajoberstar.grgit' version '1.5.0'
+  id 'java'
```

Comment:
I think `java` is implicit by `groovy`, so it should be redundant but does not hurt to add.

Reply:
I have removed it. It was suggested in one of the document I referenced. But if it is redundant, then I have removed it.